cattleshed-conf/compilers.py

```diff
@@ -1806,7 +1806,7 @@ def make_pony(self):
                 'language': 'Pony',
                 'output-file': 'prog.pony',
                 'compiler-option-raw': True,
-                'compile-command': ['/bin/bash', '-c', "mkdir -p prog && cp ./prog.pony ./prog/main.pony && /opt/wandbox/pony-{cv}/bin/ponyc ./prog"],
+                'compile-command': ['/bin/bash', '-c', "mkdir -p /tmp/prog && cp ./prog.pony /tmp/prog/main.pony && /opt/wandbox/pony-{cv}/bin/ponyc /tmp/prog"],
```

Comment:
`ponyc` outputs compiled binary as name `./prog1` (`./prog` is already existed), that is the reason why integration test fails with error `execve: Permission denied`.